src/NoFrixion.MoneyMoov/Models/Account/AccountUpdate.cs

```diff
@@ -0,0 +1,25 @@
+﻿// -----------------------------------------------------------------------------
+//  Filename: AccountUpdate.cs
+// 
+//  Description: Contains details of a account update request:
+// 
+//  Author(s):
+//  Pablo Maldonado (pablo@nofrixion.com)
+// 
+//  History:
+//  18 09 2023  Pablo Maldonado   Created, Punta Colorada, Maldonado, Uruguay.
+// 
+//  License:
+//  MIT.
+// -----------------------------------------------------------------------------
+
+namespace NoFrixion.MoneyMoov.Models;
+
+public class AccountUpdate
+{
+    public Guid UserId { get; }
```

Comment:
What does the UserId represent? When doing an update we won't trust this field. The UserID will be taken from the authenticated identity.

Reply:
You're right - It wasn't even used.

I just removed it on the [latest commit](https://github.com/nofrixion/moneymoov-dotnet/pull/189/commits/18ac0c49501a13908c47b30611978028a973ed1d)